lua/nvim-web-devicons.lua

```diff
@@ -1373,4 +1377,3 @@ return {
   has_loaded = function() return loaded end,
   get_icons = function() return icons end,
   set_up_highlights = set_up_highlights,
-}
```

Comment:
just remove that diff and we'll be good to go then :)

Reply:
Should be fixed now.